setup.py

```diff
@@ -34,7 +34,7 @@ def load_requirements(*requirements_paths):
 
 setup(
     name='edx-proctoring',
-    version='0.6.2',
+    version='0.7.2',
```

Comment:
Hi Will, this is going to run in conflict on a commit that I have on rc/2015-08-25 which bumps the version for this upcoming release. I haven't completely ironed out my release process, but I'm setting the setup.py version on the rc branches and then merging those back to master.

If you need to bump it up, can you make it 0.6.3 for now and I'll deal with the merge conflict. The version on the rc is 0.7.1